src/components/Footer/FooterSocial/FooterSocialItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import cx from 'classnames';
+
+type FooterSocialItemProps = {
+  className?: string;
+  text: string;
+  url: string;
+};
+
+export const FooterSocialItem = ({
+  className,
+  text,
+  url
+}: FooterSocialItemProps) => {
+  const classNames = cx('footer-social__icon newwindow-active', {
```

Comment:
Can we remove the `newwindow-active` classname for now? And we'll re-implement in a separate PR (giving us scope to refactor how this is done)